readme.txt

```diff
@@ -327,11 +327,12 @@ Please see the changelog for the complete list of changes in this release. Remem
 * Fix - Improved legacy URL redirect logic to prevent unwanted redirects (our thanks to wesleyanhq and Adam Schwartz for highlighting this issue) [86942]
 * Fix - Modified tribe_get_template_part() to remove potential for multiple templates to be rendered in a single call [46630]
 * Fix - Removed code which was automatically populating various address fields with default values when creating a new venue from within the event editor [44732]
+* Fix - Resolved some issues with the "Show Map" and "Show Map Link" options in CSV files not being reliably respected on import, especially when imported through Event Aggregator (props @joappf and many others for highlighting this issue) [82784]
```

Comment:
> especially when imported through Event Aggregator

This caught my eye. Isn't a CSV import a CSV import / is anything lose if we ditch that note and shorten up the changelog entry? (Worried it might confuse some folks)

Reply:
good call; fixed!